multisig-factory/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "multisig-factory"
+version = "0.1.0"
+authors = ["Near Inc <hello@near.org>"]
+edition = "2018"
+publish = false
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[profile.release]
+codegen-units = 1
+# Tell `rustc` to optimize for small code size.
+opt-level = "z"
+lto = true
+debug = false
+panic = "abort"
+overflow-checks = true
+
+[dependencies]
+near-sdk = "2.0.0"
```

Comment:
Yikes, this is old, but not as important for the factory to up to date I suppose